account_receipt_sale/wizard/sale_make_invoice.py

```diff
@@ -1,11 +1,30 @@
-from odoo import models
+from odoo import fields, models
 
 
-class SaleAdvancePaymentInv(models.TransientModel):
+class SaleAdvancePaymentRec(models.TransientModel):
+    _name = "sale.advance.payment.rec"
```

Comment:
Hi Sergio, is it necessary to re-define a new name for the model? If that's the case could you also add `_description` attribute? Otherwise it will give annoying warnings in the log during the module update

Reply:
> Hi Sergio, is it necessary to re-define a new name for the model? If that's the case could you also add `_description` attribute? Otherwise it will give annoying warnings in the log during the module update

I'd say yes, `_description` added [here](https://github.com/OCA/account-invoicing/commit/7d62986ddb7af13927ba251ef90ed0319d65219c#diff-b8cc7465a8d18dac20baa7aa762f5df2c62b39e53e49f7dd65201071dc7af4e5R6), thanks!

Reply:
Perfect, thanks!